### DISCUSSION TITLE
Sleep after hotcorner cmd

### DISCUSSION THREAD

**Discussion by thalesmaoa** [elementary/triage#206] [Q&A] [answered]
I'm using elementaryOS for my Desktop. Since I don't have a touchpad, I've configured the bottom left and bottom right commands to change workspace.

```
#!/bin/bash
wmctrl -s $(expr $(wmctrl -d | grep '*' | grep -o '^[0-9]') + 1)
```

The problem is that, sometime, hotcorner is detected again. Instead of changing to workspace 2, I jump to workspace 3 or 4. If there are some way to hold hotcorner detection, I can tweak it.

**Comment by lenemter** [comment, marked as answer]
> If there are some way to hold hotcorner detection, I can tweak it.

Unfortunately there is no way to change hotcorner detection delay. By the way we recently released a new version of the Desktop plug, which brings the ability to configure hotcorners to switch the workspaces. You can try to use it and see if it solves the issue for you :)

**Comment by thalesmaoa** [comment]
How can I add it. After update, I can't see it. Only alternate to a new workspace.

**Reply by lenemter** [reply]
Please make sure the system is fully updated through Appcenter. This is very new feature so it requires the latest version of the OS.

**Comment by thalesmaoa** [comment]
Perfect! I found it on eOS 7.
